ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/RecoveryRequired.java

```diff
@@ -0,0 +1,14 @@
+package com.arjuna.ats.jta.logging;
+
+public class RecoveryRequired {
```

Comment:
What is your reasoning for naming the class as `RecoveryRequired`?

Reply:
I thought it would add meaning to the action, that user has to take after he gets the result.